lib/database.js

```diff
@@ -21,8 +21,8 @@ exports.connect = function connect(options) {
     }
 
     if (client === 'mysql') {
-        options.connection.timezone = 'UTC';
-        options.connection.charset = 'utf8mb4';
+        options.connection.timezone = options.connection.timezone || 'UTC';
```

Comment:
This looks great 👍 
Could you please remove the @TODO in line 12/13?
Thanks :)